R/optionalInput.R

```diff
@@ -52,101 +52,99 @@
+#'   div(
+#'     optionalSelectInput(
+#'       inputId = "c4",
+#'       label = "Default",
+#'       choices = LETTERS[1:5],
+#'       selected = "A"
 #'     ),
-#'     div(
-#'       optionalSelectInput(
-#'         inputId = "c5",
-#'         label = "Named vector",
-#'         choices = c(`A - value A` = "A", `B - value B` = "B", `C - value C` = "C"),
-#'         selected = "A"
-#'       ),
-#'       verbatimTextOutput(outputId = "c5_out")
+#'     verbatimTextOutput(outputId = "c4_out")
+#'   ),
+#'   div(
+#'     optionalSelectInput(
+#'       inputId = "c5",
+#'       label = "Named vector",
+#'       choices = c(`A - value A` = "A", `B - value B` = "B", `C - value C` = "C"),
+#'       selected = "A"
 #'     ),
-#'     div(
-#'       selectInput(
-#'         inputId = "c6_choices", label = "Update choices", choices = letters, multiple = TRUE
-#'       ),
-#'       optionalSelectInput(
-#'         inputId = "c6",
-#'         label = "Updated choices",
-#'         choices = NULL,
-#'         multiple = TRUE,
-#'         fixed_on_single = TRUE
-#'       ),
-#'       verbatimTextOutput(outputId = "c6_out")
-#'     )
+#'     verbatimTextOutput(outputId = "c5_out")
 #'   ),
-#'   server = function(input, output, session) {
-#'     observeEvent(input$c6_choices, ignoreNULL = FALSE, {
-#'       updateOptionalSelectInput(
-#'         session = session,
-#'         inputId = "c6",
-#'         choices = input$c6_choices,
-#'         selected = input$c6_choices
-#'       )
-#'     })
-#'
-#'     output$c1_out <- renderPrint({
-#'       input$c1
-#'     })
-#'     output$c2_out <- renderPrint({
-#'       input$c2
-#'     })
-#'     output$c3_out <- renderPrint({
-#'       input$c3
-#'     })
-#'     output$c4_out <- renderPrint({
-#'       input$c4
-#'     })
-#'     output$c5_out <- renderPrint({
-#'       input$c5
-#'     })
-#'     output$c6_out <- renderPrint({
-#'       input$c6
-#'     })
-#'   }
+#'   div(
+#'     selectInput(
+#'       inputId = "c6_choices", label = "Update choices", choices = letters, multiple = TRUE
+#'     ),
+#'     optionalSelectInput(
+#'       inputId = "c6",
+#'       label = "Updated choices",
+#'       choices = NULL,
+#'       multiple = TRUE,
+#'       fixed_on_single = TRUE
+#'     ),
+#'     verbatimTextOutput(outputId = "c6_out")
+#'   )
 #' )
 #'
+#' server <- function(input, output, session) {
+#'   observeEvent(input$c6_choices, ignoreNULL = FALSE, {
+#'     updateOptionalSelectInput(
+#'       session = session,
+#'       inputId = "c6",
+#'       choices = input$c6_choices,
+#'       selected = input$c6_choices
+#'     )
+#'   })
+#'
+#'   output$c1_out <- renderPrint({
+#'     input$c1
+#'   })
+#'   output$c2_out <- renderPrint({
+#'     input$c2
+#'   })
+#'   output$c3_out <- renderPrint({
+#'     input$c3
+#'   })
+#'   output$c4_out <- renderPrint({
+#'     input$c4
+#'   })
+#'   output$c5_out <- renderPrint({
+#'     input$c5
+#'   })
+#'   output$c6_out <- renderPrint({
+#'     input$c6
+#'   })
```

Comment:
How about compacting this a bit?

```suggestion
#'   output$c1_out <- renderPrint(input$c1)
#'   output$c2_out <- renderPrint(input$c2)
#'   output$c3_out <- renderPrint(input$c3)
#'   output$c4_out <- renderPrint(input$c4)
#'   output$c5_out <- renderPrint(input$c5)
#'   output$c6_out <- renderPrint(input$c6)
```